docs/static/libs/algolia/algolia-mobile.css

```diff
@@ -7,25 +7,26 @@
     margin-bottom: 0px;
   }
 
-  .algolia-autocomplete.algolia-autocomplete-left {
-    left: 0px !important;
+  .algolia-autocomplete.algolia-autocomplete-right .ds-dropdown-menu {
+    left: -16px !important;
```

Comment:
Are these `!important` flags still necessary? (they might be, I'm just unsure why)

Reply:
Just double-checked and they are necessary -- the `algolia.min.css` was pretty difficult to override:

<img width="659" alt="Screen Shot 2020-07-20 at 9 05 32 PM" src="https://user-images.githubusercontent.com/56043296/88000883-c118f980-cacc-11ea-977c-7688e423e339.png">

This might be how `!important` works or how overridding works and I'd love to learn more --- but if I don't have _both_ of these lines, inspectElement would show my media queries crossed out (and `algolia.min.css` taking precedence)

Reply:
For reference (or `ref`, haha), `algolia.min.css` has this property that I had to override:

<img width="641" alt="Screen Shot 2020-07-20 at 9 14 47 PM" src="https://user-images.githubusercontent.com/56043296/88001430-23262e80-cace-11ea-941e-8c95c7b064e4.png">

Reply:
Ah interesting, it looks like we are a bit behind on our algolia styling because we copy/pasted it from the source and are now running our own version. (The CSS version we should be using is [here](https://cdn.jsdelivr.net/npm/docsearch.js@2.5.2/dist/cdn/docsearch.min.css)). --This is another ticket/task (to add these new styles and edit our overrides).

Also, since the source we are using uses `!important`, it forces us to as well. 

Thanks for digging into this!